docs/docs/02-guide/06-ibc.md

```diff
@@ -526,70 +526,27 @@ found` and no action is taken.
 
 ### Configure and start the relayer
 
-First, configure the relayer. Use the Ignite CLI `configure` command with the
-`--advanced` option:
+First, add the Hermes relayer app.
 
 ```bash
-ignite relayer configure -a \
-  --source-rpc "http://0.0.0.0:26657" \
-  --source-faucet "http://0.0.0.0:4500" \
-  --source-port "blog" \
-  --source-version "blog-1" \
-  --source-gasprice "0.0000025stake" \
-  --source-prefix "cosmos" \
-  --source-gaslimit 300000 \
-  --target-rpc "http://0.0.0.0:26659" \
-  --target-faucet "http://0.0.0.0:4501" \
-  --target-port "blog" \
-  --target-version "blog-1" \
-  --target-gasprice "0.0000025stake" \
-  --target-prefix "cosmos" \
-  --target-gaslimit 300000
-```
-
-When prompted, press Enter to accept the default values for `Source Account` and
-`Target Account`.
-
-The output looks like:
-
-```
----------------------------------------------
-Setting up chains
----------------------------------------------
-
-🔐  Account on "source" is "cosmos1xcxgzq75yrxzd0tu2kwmwajv7j550dkj7m00za"
-
- |· received coins from a faucet
- |· (balance: 100000stake,5token)
-
-🔐  Account on "target" is "cosmos1nxg8e4mfp5v7sea6ez23a65rvy0j59kayqr8cx"
-
- |· received coins from a faucet
- |· (balance: 100000stake,5token)
-
-⛓  Configured chains: earth-mars
+ignite app install -g github.com/ignite/apps/hermes
 ```
 
-In a new terminal window, start the relayer process:
+and after configure the relayer.
 
 ```bash
-ignite relayer connect
+ignite relayer hermes configure \
+"earth-1" "http://localhost:26657" "http://localhost:9090" --chain-a-faucet "http://0.0.0.0:4500" \
```

Comment:
Is "earth-1" supposed to be chain-id? Because I think it's just `earth` in this case